frontend/packages/core/src/AppLayout/drawer.tsx

```diff
@@ -1,198 +1,213 @@
-  ${({ theme }) => `
-  background-color: ${theme.palette.accent.main};
-  `}
-`;
+// sidebar
+const drawerWidth = "100px";
+
+const DrawerPanel = styled(MuiDrawer)({
+  flexShrink: 0,
+  width: `${drawerWidth}`,
+  "div[class*='MuiDrawer-paper']": {
+    width: `${drawerWidth}`,
+    top: "64px",
+    backgroundColor: "linear-gradient(0deg, #FFFFFF, #FFFFFF), #FFFFFF",
+    boxShadow: "0px 5px 15px rgba(53, 72, 212, 0.2)",
+  },
+});
+
+// sidebar groupings
+const GroupList = styled(List)({
+  padding: "0px",
+});
+
+const GroupListItem = styled(ListItem)({
+  flexDirection: "column",
+  height: "82px",
+  padding: "16px 8px 0px 8px",
+  "&:hover": {
+    backgroundColor: "#E7E7EA",
+  },
+  "&:active": {
+    backgroundColor:
+      "linear-gradient(0deg, rgba(255, 255, 255, 0.85), rgba(255, 255, 255, 0.85)), #0D1030",
+  },
+  "&.Mui-selected": {
+    backgroundColor: "#EBEDFB",
+    ".MuiAvatar-root": {
+      color: "#FFFFFF",
+      background: "#3548D4",
+    },
+  },
+});
+
+const Avatar = styled(MuiAvatar)({
+  color: "rgba(13, 16, 48, 0.38)",
+  background: "rgba(13, 16, 48, 0.12)",
+  height: "36px",
+  width: "36px",
+});
+
+const GroupHeading = styled(Typography)({
+  color: "rgba(13, 16, 48, 0.6)",
+  fontWeight: 500,
+  fontSize: "14px",
+  lineHeight: "18px",
+  flexGrow: 1,
+  paddingTop: "8px",
+  paddingBottom: "8px",
+});
+
+// sidebar submenu
+const Popper = styled(MuiPopper)({
+  zIndex: 1200,
+  paddingTop: "16px",
+});
+
+const Paper = styled(MuiPaper)({
+  minWidth: "230px",
+  border: "1px solid #E7E7EA",
+  boxShadow: "0px 10px 24px rgba(35, 48, 143, 0.3)",
+});
+
+// sudebar submenu groupings
+const LinkListItem = styled(ListItem)({
+  backgroundColor: "#FFFFFF",
+  "&.Mui-selected": {
+    backgroundColor: "#EBEDFB",
+    ".MuiTypography-root": {
+      color: "#3548D4",
+    },
+  },
+  "&:hover": {
+    backgroundColor: "#E7E7EA",
+  },
+  height: "48px",
+});
+
+const LinkHeading = styled(Typography)({
+  color: "rgba(13, 16, 48, 0.6)",
+  fontWeight: 500,
+  fontSize: "14px",
+  lineHeight: "18px",
+});
 
 interface GroupProps {
   heading: string;
   open: boolean;
   updateOpenGroup: (heading: string) => void;
-  onNavigate: () => void;
 }
 
-const Group: React.FC<GroupProps> = ({
-  heading,
-  open = false,
-  updateOpenGroup,
-  onNavigate,
-  children,
-}) => {
+const Group: React.FC<GroupProps> = ({ heading, open = false, updateOpenGroup, children }) => {
   const childrenList = React.Children.toArray(children);
 
+  const [openList, setListOpen] = React.useState(false);
+
+  const anchorRef = React.useRef(null);
+
+  const handleToggle = () => {
+    setListOpen(!openList);
+  };
+
+  const handleClose = event => {
+    if (anchorRef.current && anchorRef.current.contains(event.target)) {
+      return;
+    }
+    setListOpen(false);
+  };
+
+  let headingPath = window.location.pathname.replace("/", "").split("/")[0];
+
+  if (headingPath === "ec2") {
+    headingPath = "aws";
+  }
```

Comment:
this isn't going to be easy to maintain, we'll need a TODO to discover this from config or something.